OpenSim/Simulation/Test/testFrames.cpp

```diff
@@ -441,7 +441,7 @@ void testFilterByFrameType()
             << " of type " << typeid(component).name() << std::endl;
     }
 
-    ASSERT_EQUAL(9, i, __FILE__, __LINE__,
+    ASSERT_EQUAL(11, i, __FILE__, __LINE__,
```

Comment:
Suggest changing `9` to `11` in the comment (L445) as well (to avoid mismatch between condition and message). Analogous comment applies to L453 and L474.